### DISCUSSION TITLE
Build images from python:3.11-slim-bookworm

### DISCUSSION THREAD

**Discussion by ondrejmo** [FlareSolverr/FlareSolverr#1205] [Ideas]
The bullseye based images are full of unnecessary vulnerabilities, it would be nice to update them to bookworm. I tested it locally in my environment, everything worked as expected.

### Screenshot from Trivy results

![image](https://github.com/FlareSolverr/FlareSolverr/assets/11840034/f6942c65-0c81-4e61-8ea8-545287e217f5)

**Comment by ilike2burnthing** [comment]
There are a few PRs and contributor images which include this, should be implemented soon.
